lighthouse-core/gather/driver.js

```diff
@@ -438,16 +438,19 @@ class Driver {
   }
 
   /**
+   * Note: Prefer `evaluate` instead.
    * Evaluate an expression in the context of the current page. If useIsolation is true, the expression
    * will be evaluated in a content script that has access to the page's DOM but whose JavaScript state
    * is completely separate.
    * Returns a promise that resolves on the expression's value.
+   * @template T
    * @param {string} expression
    * @param {{useIsolation?: boolean}=} options
    * @return {Promise<*>}
    */
   async evaluateAsync(expression, options = {}) {
-    const contextId = options.useIsolation ? await this._getOrCreateIsolatedContextId() : undefined;
+    const contextId =
```

Comment:
revert?

Reply:
weird, it's at 101 columns but yarn lint doesn't flag it right now..